scene/gui/color_picker.cpp

```diff
@@ -563,7 +563,7 @@ void ColorPicker::_html_submitted(const String &p_html) {
 		color.a = previous_color.a;
 	}
 
-	if (color == previous_color) {
+	if (color.to_abgr32() == previous_color.to_abgr32()) {
```

Comment:
I think this check may be too late here. At this point, the internal `color` state (that's a class member, not a local variable) has already been overwritten with the return value of `Color::from_string`, so this only skips the change notification, doesn't it?

Reply:
Hmm, you're right.